libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -85,7 +85,6 @@
 static NSString * const kSFOAuthGrantTypeAuthorizationCode      = @"authorization_code";
 static NSString * const kSFOAuthResponseTypeActivatedClientCode = @"activated_client_code";
 static NSString * const kSFOAuthResponseClientSecret            = @"client_secret";
```

Comment:
I'm 100% sure that flow can go away, given that nothing sets the activationCode anymore, other than code to revoke it, which itself is never called either.  FWIW. 😊